src/state_machine/coordinator/mod.rs

```diff
@@ -1117,19 +1117,23 @@ pub mod test {
             }
         }
 
+        assert_eq!(outbound_messages.len(), 1);
```

Comment:
This `assert!` is a duplicate of the one on line `1112`.

Reply:
Oh right, will remove.